### PR TITLE
Switch container size metric from physical to logical capacity

### LINKED ISSUE #47: Switch containers size to count logical objects rather instead of physical
Switching frostfs_node_engine_container_size to counting sizes of logical objects instead of physical will improve future integrations with billing systems.

### PR DESCRIPTION
**Pull request by ir0nbee**
Closes #47

### CODE REVIEW

File: cmd/frostfs-node/object.go [hunk 1]
```diff
@@ -316,7 +316,7 @@ func initObjectService(c *cfg) {
 		deletesvc.WithPutService(sPut),
 		deletesvc.WithNetworkInfo(&delNetInfo{
 			State:      c.cfgNetmap.state,
-			tsLifetime: 5,
+			tsLifetime: 1,
```
Comment (alexvanin) [inline cmd/frostfs-node/object.go:319]: Looks like debug change.

**Review by acid-ant — APPROVED** [review]

File: pkg/local_object_storage/metabase/inhume.go [hunk 2]
```diff
@@ -164,10 +186,12 @@ func (db *DB) Inhume(prm InhumePrm) (res InhumeRes, err error) {
 			obj, err := db.get(tx, prm.target[i], buf, false, true, currEpoch)
 			targetKey := addressKey(prm.target[i], buf)
 			if err == nil {
+				containerID, _ := obj.ContainerID()
 				if inGraveyardWithKey(targetKey, graveyardBKT, garbageBKT) == 0 {
-					// object is available, decrement the
+					// object is available, increment the
 					// logical counter
 					inhumed++
```
Comment (fyrchik) [inline pkg/local_object_storage/metabase/inhume.go:193]: Can we remove this comment? It is (was) misleading `logical counter` shouldn't increase at all on removal, what we increase is some other counter.

File: pkg/local_object_storage/metabase/inhume.go [hunk 3]
```diff
@@ -164,10 +186,12 @@ func (db *DB) Inhume(prm InhumePrm) (res InhumeRes, err error) {
 			obj, err := db.get(tx, prm.target[i], buf, false, true, currEpoch)
 			targetKey := addressKey(prm.target[i], buf)
 			if err == nil {
+				containerID, _ := obj.ContainerID()
```
Comment (fyrchik) [inline pkg/local_object_storage/metabase/inhume.go:189]: Why have you decided to have it outside `if`?

File: pkg/local_object_storage/metabase/inhume.go [hunk 4]
```diff
@@ -42,6 +45,25 @@ func (i InhumeRes) DeletedLockObjects() []oid.Address {
...
+// associated container ID by index.
+func (i InhumeRes) GetDeletionInfoByIndex(target int) (cid.ID, uint64) {
+	return i.containerIds[target], i.sizes[target]
+}
+
+// StoreDeletionInfo stores size of deleted object and associated container ID
+// in corresponding arrays.
+func (i *InhumeRes) StoreDeletionInfo(containerID cid.ID, deletedSize uint64) {
+	i.sizes = append(i.sizes, deletedSize)
+	i.containerIds = append(i.containerIds, containerID)
```
Comment (fyrchik) [inline pkg/local_object_storage/metabase/inhume.go:64]: We use and append them together, why not store them together? (define a private struct, return 2 values as you already do).

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ Changelog for FrostFS Node
 - Add command `frostfs-adm morph netmap-candidates` (#1889)
 
 ### Changed
+- Change `frostfs_node_engine_container_size` to counting sizes of logical objects
```
[the remark this answers is not in the document]
Comment (ir0nbee) [reply CHANGELOG.md:14]: Change {subject} to {doing something}. Looks correct to me.

File: pkg/local_object_storage/metabase/inhume.go (hunk 3 above)
Comment (ir0nbee) [reply pkg/local_object_storage/metabase/inhume.go:189]: It is inside the `if` now.

Comment (ir0nbee) [reply pkg/local_object_storage/metabase/inhume.go:189]: I mean, it always was inside the `if`. Do you mean something else?

**Review by alexvanin — APPROVED** [review]

**Pull request merged**